python/ray/air/config.py

```diff
@@ -702,8 +706,10 @@ class RunConfig:
     Args:
         name: Name of the trial or experiment. If not provided, will be deduced
             from the Trainable.
-        local_dir: Local dir to save training results to.
-            Defaults to ``~/ray_results``.
+        storage_path: Path to store results at. Can be a local directory or
+            a destination on cloud storage. If Ray storage is set up,
```

Comment:
link to how to set it up?

Reply:
Actually, removed the ray storage comment for now until we actually introduce this in a follow-up PR